src/Type/Object/PostObject.php

```diff
@@ -290,6 +290,13 @@ public static function get_post_object_fields( $post_type_object, $type_registry
 			];
 		}
 
+		if ( 'post' === $post_type_object->name ) {
```

Comment:
I haven't used sticky posts before, is "post" the only post type that supports it?

Reply:
@TylerBarnes it is.

Reply:
Cool, good to know. I guess another WP peculiarity :p

Reply:
ooh. . .what happens if someone wants to sell Cinnamon Rolls on their WordPress site. . .what does `isSticky` mean in that case? 🤔

Reply:
`isReallySticky`

Reply:
`isPhysicallySticky`?